src/data/dialogmote/useEndreReferat.ts

```diff
@@ -3,8 +3,10 @@ import { ISDIALOGMOTE_ROOT } from "@/apiConstants";
 import { NewDialogmoteReferatDTO } from "@/data/dialogmote/types/dialogmoteReferatTypes";
 import { post } from "@/api/axios";
 import { dialogmoterQueryKeys } from "@/data/dialogmote/dialogmoteQueryHooks";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
 
-export const useEndreReferat = (fnr: string, dialogmoteUuid: string) => {
+export const useEndreReferat = (dialogmoteUuid: string) => {
+  const fnr = useValgtPersonident();
```

Comment:
Disse hookene kan bruke `useValgtPersonident()` i stedet for at `fnr` må sendes inn der de brukes.